terminology/src/main/java/au/csiro/pathling/fhir/TerminologyClient.java

```diff
@@ -175,6 +175,26 @@ Parameters lookup(
       @Nullable @OperationParam(name = "property") CodeType property
   );
 
+  /**
+   * @param system the system of the code
+   * @param version the version of the code system
+   * @param code the code to lookup
+   * @param property the property or properties to be returned in the response
+   * @param acceptLanguage the preferable language of the return
+   * @return a {@link Parameters} resource
+   * @see <a
+   * href="https://www.hl7.org/fhir/R4/codesystem-operation-lookup.html">CodeSystem/$lookup</a>
+   */
+  @Operation(name = "$lookup", type = CodeSystem.class, idempotent = true)
+  @Nonnull
+  Parameters lookup(
+      @Nonnull @OperationParam(name = "system") UriType system,
+      @Nullable @OperationParam(name = "version") StringType version,
+      @Nonnull @OperationParam(name = "code") CodeType code,
+      @Nullable @OperationParam(name = "property") CodeType property,
+      @Nullable @OperationParam(name = "acceptLanguage") CodeType acceptLanguage
+  );
+  
```

Comment:
Similar to the previous comment, I am not sure that we need overloads here. It's ok to have nullable parameters and handle that nullability within the implementation, as is done with translate.